packages/client/src/components/Game/Game.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useRef } from 'react';
+import Game from '../../logic/Game/Game';
+
+function GameComponent() {
```

Comment:
В проекте принято использовать стрелочные функции:
```suggestion
const GameComponent = () => {
```